.travis.yml

```diff
@@ -1,13 +1,17 @@
 language: python
 sudo: false
 cache: pip
-dist: xenial
+dist: bionic
 matrix:
   include:
     - python: "3.6"
+      env: TOXENV=py36,mypy
```

Comment:
Glad to see we got the syntax down! That being said, I don't think we actually need to run mypy four times; shouldn't once be enough? And could we put it in its own toxenv so it's the only thing being called for that test run? This should look like adding, for example:


```
    - python: "3.9"
      env: TOXENV=mypy
```

Reply:
OK :+1: